src/tree/SubscriptionTreeItem.ts

```diff
@@ -90,7 +90,11 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
             ext.outputChannel.appendLog(createMessage);
             const server = nonNullProp(wizardContext, 'server');
             const host = nonNullProp(server, 'fullyQualifiedDomainName');
-            const username: string = nonNullProp(wizardContext, 'adminUser');
+            let username: string = nonNullProp(wizardContext, 'adminUser');
```

Comment:
You're trying to fix a symptom of the problem, rather than the root cause. This is not the first time you've moved these few lines of code because they weren't executed at the right time. I still don't think it's right because I believe `PostgresServerSetCredentialsStep` needs the username _with_ the suffix.

Try to come up with a solution that addresses the root cause - something that makes these kinds of bugs less likely to happen in the future. It will probably require some sort of renaming or refactoring.

Reply:
That's true. `createAzureClient` in the PostgresServerCreatStep is the only place we need to use username without the suffix, and anything after that will require the full username including the suffix.